engine/session.js

```diff
@@ -582,14 +582,8 @@ class Session {
         await timer(500);
         playheadState = await this._playheadState.getValues(["mediaSeq", "vodMediaSeqVideo"]);
       } else {
-        debug(`[${this._sessionId}]: Recently Loaded Next Vod. PlayheadState not up-to-date. Return the last generated m3u8`);
-        const m3u8 = await this._playheadState.getLastM3u8();
```

Comment:
was not a reliable m3u8 when in HA mode.   Other option would be to store a lastm3u8 for video and audio in store.